File: Hexa.Business/Models/Catalog/CategoryModel.cs
using Hexa.Business.Models.Shared;

namespace Hexa.Business.Models.Catalog
{
    public class CategoryModel : BaseModel
    {
        public string Name { get; set; }

        public string Description { get; set; }

        public string PictureId { get; set; }

        public int ParentCategoryId { get; set; }

        public bool IncludeInNavigation { get; set; }

        public bool Active { get; set; }

        public bool Deleted { get; set; }

        public int DisplayOrder { get; set; }
    }
}

File: HexaCommerce/ClientApp/app/modules/admin/components/category/category.model.ts
export class CategoryModel {
    public Id: number;
    public Name: string;
    public Description: string;
    public PictureUrl: string;
    public ParentCategoryId: number;
    public IncludeInNavigation: boolean;
    public Active: boolean;
    public Deleted: boolean;
    public DisplayOrder: number;
    public CreatedOn: Date;
}

